### PR TITLE
Creat Private Events app

### PR DESCRIPTION
**Pull request by petudeveloper**

### CODE REVIEW

File: app/views/events/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                    <span><strong>Date: </strong></span>
+                    <small><%= event.date %></small>
+                </p>
+            </li>
+            <p><%= link_to "Show event details", event %></p>
+        <% end %>
+        </ul>
+        <h5>Past Events</h5>
+        <ul>
+        <% current_user.attended_events.past_events.find_each do |event| %>
```
Comment (cyonii) [inline app/views/events/index.html.erb]: #### MANDATORY
- [x] You should list past events, i.e. `@past_events = Event.past_events` in your `#index` action and use `@past_events` in your view ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/past_upcoming_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

File: app/views/events/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        <% end %>
+    </div>
+</div>
+<div>
+    <h2 class="text-center text-danger my-5">Invitations</h2>
+
+    <div class="d-flex flex-column justify-content-center align-items-center">
+        <h5>Upcoming Events</h5>
+        <ul>
+        <% current_user.attended_events.future_events.find_each do |event| %>
```
Comment (cyonii) [inline app/views/events/index.html.erb]: #### MANDATORY
- [x] You should list upcoming events, i.e. `@upcoming_events = Event.future_events` in your event's `#index` action and use `@upcoming_events ` in your view ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/past_upcoming_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

File: app/views/events/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,65 @@
+<% if user_signed_in? %>
+<div>
+    <h2 class="text-center text-danger my-5">Your Events</h2>
+    <div class="d-flex flex-column justify-content-center align-items-center">
+        <% current_user.events.each do |event| %>
+            <ul>
+                <p class="text-center"><strong class="text-primary"><%= event.title %></strong></p>
```
Comment (cyonii) [inline app/views/events/index.html.erb]: #### MANDATORY
- [x] To list events created by user, i.e. `@created_events = current_user.events`, kindly define this as an instance variable in your controller and use the variable in your view. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/current_user_show_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

**Review by cyonii — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED ♻️✍️

Hello team, you have a great project so far, well done.

#### TO HIGHLIGHT
- Good README

#### TO IMPROVE (MANDATORY)
- [x] I ran into a problem with webpack while setting up your project, kindly include the command to update webpack in your README. `yarn install`
- [x] Navigation links should be present at the top of the page to make it easier to jump around the page ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/navigation_links_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))
- [x] There're typos in your commit messages, you misspelt the word **"Create"** as **"Creat"** a few times. Please make sure to avoid small defects like this, and keep your projects as professional as can be.
- [x] Event's index page displays all events separated by past and upcoming events lists  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/past_upcoming_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))
- [x] The user's show page should display events the user has created and events that the user is attending, separated in past and upcoming ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations)). Currently, you do not have a user show page, please implement this.

> #### <br>Feel free to leave a comment or question if something is not clear.<br> Slack: @[CY Kalu](https://microverse-students.slack.com/team/U01494U96FR)<br><br>

**Comment by petudeveloper** [comment]
@cyonii Thanks for your comments and feedback.  We followed all your suggestions:

- [x] Include the command to update webpack ```yarn install```  in the README file.
- [x] Create a navigation bar at the top of the page and add the navigation links there.
- [x] Modify index page to display events separated by past and upcoming events lists.
- [x] Create user's show page and set it up to display events the user has created and events that the user is attending.

other comments:
- [x]  Define this```@past_events = Event.past_events```, ```@upcoming_events = Event.future_events``` and ```@created_events = current_user.events``` as an instance variable in the corresponding controllers.

**Comment by cyonii** [comment]
Great effort there 🙃💪🏼

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
# OPTIONAL
- [ ] Views should be completely separate from app logic. Please move all view-related logic to app/helpers.
### Happy Coding ☺️
## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

**Pull request merged**